### PR TITLE
Fix complex/yul instructions staticcall

### PR DESCRIPTION
**Pull request by MarcosNicolau**
They were failing because we were not correctly panicking when checking static mode.

### CODE REVIEW

**Comment by MarcosNicolau** [comment]
depends on #152

**Review by jorbush — APPROVED** [review]
LGTM

**Review by gianbelinche — APPROVED** [review]
LGTM

File: src/lib.rs [hunk 1]
```diff
@@ -117,14 +117,15 @@ pub fn run(
...
+        let can_execute = vm.can_execute(&opcode);
+        if vm.decrease_gas(opcode.gas_cost).is_err() || can_execute.is_err() {
             match inexplicit_panic(&mut vm, storage) {
                 Ok(false) => continue,
                 _ => return Ok((ExecutionOutput::Panic, vm)),
             }
         }
 
-        if vm.can_execute(&opcode)? {
+        if can_execute.unwrap() {
```
Comment (juan518munoz) [inline src/lib.rs:128]: Handle error:
```suggestion
        if can_execute? {
```

**Review by juan518munoz — CHANGES_REQUESTED** [review]

**Comment by juan518munoz** [comment]
Ignore requested changes, I missed the part above where `is_err()` is checked, so you don't need to `?` the `Result`.

You can still apply the suggestion to prevent having `unwrap`s on our codebase, but it's up to you.

File: src/lib.rs (hunk 1 above)
Comment (MarcosNicolau) [reply src/lib.rs:128]: In theory, it shouldn't return err since we are already checking if is_err above, but I'll change it just to make sure

**Review by juan518munoz — APPROVED** [review]

**Pull request merged**